.github/workflows/smartbear-issue-comment-created.yml

```diff
@@ -0,0 +1,60 @@
+name: Issue Comment Created
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  jira:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.comment.body == '/jira ticket' }}
+    steps:
+    - run: echo ${{ github.event.comment.body }}
+    
+    - name: Login
+      uses: atlassian/gajira-login@v3
+      env:
+        JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+        JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+        JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+        ISSUE_TITLE: ${{ github.event.issue.title }}
+    
+    - name: SearchParam
+      run: echo 'summary ~ ${{ toJSON($ISSUE_TITLE)}} AND project=${{ secrets.JIRA_PROJECT }}'
```

Comment:
Here's the updated search that uses the github issue number https://github.com/pact-foundation/.github/blob/master/.github/workflows/smartbear-issue-label-added.yml#L22